docs/types.rst

```diff
@@ -133,20 +133,44 @@ As a rule of thumb, use `bytes` for arbitrary-length raw byte data and `string`
 for arbitrary-length string (utf-8) data. If you can limit the length to a certain
 number of bytes, always use one of `bytes1` to `bytes32` because they are much cheaper.
 
+.. index:: ! ufixed, ! fixed, ! fixed point number
+
+Fixed Point Numbers
+-------------------
+
+**bold** COMING SOON... **bold**
+
 .. index:: literal, literal;integer
 
 .. _integer_literals:
 
 Integer Literals
------------------
+ ------------------
 
 Integer Literals are arbitrary precision integers until they are used together with a non-literal. In `var x = 1 - 2;`, for example, the value of `1 - 2` is `-1`, which is assigned to `x` and thus `x` receives the type `int8` -- the smallest type that contains `-1`, although the natural types of `1` and `2` are actually `uint8`.    
-
+  
 It is even possible to temporarily exceed the maximum of 256 bits as long as only integer literals are used for the computation: `var x = (0xffffffffffffffffffff * 0xffffffffffffffffffff) * 0;` Here, `x` will have the value `0` and thus the type `uint8`.
 
+.. index:: literal, literal;rational
+
+.. _rationals:
+
+Rationals
```

Comment:
I think the term `literal` should also appear in the heading here. What about `Integer and Rational Literals`?